src/benchmark_environments/classic.py

```diff
@@ -0,0 +1,32 @@
+"""Adaptation of classic Gym environments for IRL."""
+from gym.wrappers import TimeLimit
+
+from benchmark_environments import util
+
+register = util.curried_gym_register_as_decorator(__name__)
+
+
+@register("CartPole-v0")
+def cart_pole():
```

Comment:
`cart_pole` and `mountain_car` are identical functions except for the environment names. Maybe make a functor? e.g.

```
def fixed_horizon(env_name):
    def make_env():
         env = util.make_env_no_wrappers(env_name)
         # ...
    return make_env

# ...

# If you never actually use register as a decorator, maybe change it to be register(our_name, env_thunk_fn) rather than register(our_name)(env_thunk_fn)
register("CartPole-v0")(fixed_horizon("CartPole-v1")
```

Reply:
Ah I see they differ in sign of the final reward, but that seems easy to add as a parameter.

Reply:
I don't think the functor is worth making until there is more duplication (IE, at least three functions of this type or longer functions duplicated).

I removed the "automatically detect the time limit of cartpole/mountaincar" logic so that the code can match up with the docstring, and this reduces the number of lines duplicated.